system/Database/MigrationRunner.php

```diff
@@ -817,9 +817,8 @@ public function ensureTable()
      */
     protected function migrate($direction, $migration): bool
     {
-        include_once $migration->path;
-
-        $class = $migration->class;
+        $class = $this->generateUniqClassname($migration);
```

Comment:
Person preference: I'd rather see this spelled out than save two characters:
```suggestion
        $class = $this->generateUniqueClassname($migration);
```